releasenotes/notes/go1.16.7-4ec8477608022a26.yaml

```diff
@@ -0,0 +1,16 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Agents are now built with Go1.16.  This will have one user-visible change:
+    on Linux, the process-level RSS metric for agent processes will be
+    reduced from earlier versions.  This reflects a change in how memory
+    usage is calculated, not a reduction in used memory, and is an artifact
+    of the Go runtime `switching from MADV_FREE to MADV_DONTNEED
+    <https://golang.org/doc/go1.16#runtime>`_.
```

Comment:
```suggestion
    Agents are now built with Go 1.16.  This has one user-visible change:
    on Linux, the process-level RSS metric for Agent processes is
    reduced from earlier versions.  This reflects a change in how memory
    usage is calculated, not a reduction in used memory, and is an artifact
    of the Go runtime switching from `MADV_FREE` to `MADV_DONTNEED`
    <https://golang.org/doc/go1.16#runtime>`.
```

Reply:
This is a reStructuredText document, not markdown, so the proposed change to the backticks means that only `MADV_DONTNEED` gets linked, and not the whole phrase.  Is that the intent?